packages/rich-text-editor/package.json

```diff
@@ -29,32 +29,18 @@
   "private": false,
   "license": "MIT",
   "dependencies": {
-    "@cultureamp/rich-text-toolkit": "^1.11.0",
+    "@cultureamp/rich-text-toolkit": "^2.0.0",
     "@kaizen/a11y": "^1.6.4",
     "@kaizen/component-base": "^1.1.1",
     "@kaizen/component-library": "^16.1.1",
     "@kaizen/draft-form": "^8.3.1",
     "@kaizen/draft-tooltip": "^5.4.4",
     "@kaizen/notification": "^0.11.7",
     "classnames": "^2.3.1",
-    "prosemirror-commands": "^1.2.2",
-    "prosemirror-history": "^1.2.0",
-    "prosemirror-inputrules": "^1.1.3",
-    "prosemirror-keymap": "^1.1.5",
-    "prosemirror-model": "^1.16.1",
-    "prosemirror-schema-list": "^1.1.6",
-    "prosemirror-state": "^1.3.4",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@types/classnames": "^2.3.0",
-    "@types/prosemirror-commands": "^1.0.4",
```

Comment:
All the @type have been removed because they no longer are required in the latest ProseMirror update, which transitioned to Typescript